core/arch/arm/include/mm/generic_ram_layout.h

```diff
@@ -41,10 +41,10 @@
  * TEE_SHMEM_START	Non-secure static shared memory physical base address
  * TEE_SHMEM_SIZE	Non-secure static shared memory byte size
  *
- * TZDRAM_BASE		Main/external secure RAM base address
- * TZDRAM_SIZE		Main/external secure RAM byte size
- * TZSRAM_BASE		On-chip secure RAM base address, required by pager.
- * TZSRAM_SIZE		On-chip secure RAM byte size, required by pager.
+ * TRUSTED_DRAM_BASE	Main/external secure RAM base address
+ * TRUSTED_DRAM_SIZE	Main/external secure RAM byte size
+ * TRUSTED_SRAM_BASE	On-chip secure RAM base address, required by pager.
+ * TRUSTED_SRAM_SIZE	On-chip secure RAM byte size, required by pager.
```

Comment:
(commit "core: arm: generic_ram_layout.h: document TRUSTED_{D,S}RAM_*")
This header file does not produce these macros.

Reply:
OK, I'll drop the commit.